src/datastore/postgres-store.ts

```diff
@@ -2259,6 +2260,28 @@ export class PgDataStore
         `,
         [blockIdentifer.height]
       );
+    } else if ('burnBlockHash' in blockIdentifer) {
+      result = await client.query<BlockQueryResult>(
+        `
+        SELECT ${BLOCK_COLUMNS}
+        FROM blocks
+        WHERE burn_block_hash = $1 
+        ORDER BY canonical DESC, block_height DESC
+        LIMIT 1
+        `,
+        [blockIdentifer.burnBlockHash]
```

Comment:
I think the test failure should be fixed by changing this to:
```ts
[hexToBuffer(blockIdentifer.burnBlockHash)]
```

Reply:
Thank you for the pointer. It worked.